pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -265,32 +265,20 @@ export class Dialog extends React.Component {
     }
 
     render() {
-        var backdrop;
-        if (!this.props.no_backdrop) {
-            backdrop = <div className="modal-backdrop fade in" />;
-        }
         return (
-            <div>
-                { backdrop }
-                <div className="modal fade in dialog-ct-visible" tabIndex="-1">
-                    <div id={this.props.id} className="modal-dialog">
-                        <div className="modal-content">
-                            <div className="modal-header">
-                                <h4 className="modal-title">{ this.props.title }</h4>
-                            </div>
-                            { this.props.body }
-                            { this.props.footer }
-                        </div>
-                    </div>
-                </div>
-            </div>
+            <Modal id={this.props.id} show animation={false}>
```

Comment:
animation={false} is probably the same like omitting it, but no need for removal. Just for clarification is that true?

Reply:
No, `animation=false` is the default.

Reply:
Dang, `animation=true` is the default.  Sorry.